src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -224,4 +230,13 @@ public void setCompatibilityMode(final CompatibilityMode compatibilityMode) {
     public void setValidationStrategies(final List<ValidationStrategyConfiguration> validationStrategies) {
         this.validationStrategies = validationStrategies;
     }
+
+    @Nullable
```

Comment:
Ok, you are probably right, it's better to stay consistent here. I will remove this `JsonInclude.Include.NON_NULL`